templates/components/itilobject/timeline/form_task.html.twig

```diff
@@ -365,10 +365,16 @@
                         }
                      </script>
                      <div class="col-12">
-                        <button id="plan{{ rand }}" class="btn btn-outline-secondary d-block mb-2 text-truncate" onclick="showPlanUpdate{{ rand }}()" type="button">
+                        <button id="plan{{ rand }}" class="btn btn-outline-secondary mb-2 text-truncate" onclick="showPlanUpdate{{ rand }}()" type="button">
                            <i class="fas fa-calendar"></i>
                            <span>{{ __('Plan this task') }}</span>
                         </button>
+                        {% if subitem.can(id, constant('UPDATE')) %}
+                           <button class="btn btn-outline-danger me-2" type="submit" name="unplan"
+                                 onclick="return confirm('{{ __('Confirm the deletion of planning?') }}');">
```

Comment:
Is'nt there a better way than using `@onclick`?

Reply:
for a confirm, i think it's the best way

Reply:
:+1:  then